glusterd2/commands/volumes/volume-option.go

```diff
@@ -113,13 +134,25 @@ func xlatorAction(c transaction.TxnCtx, txnOp txnOpType, volOp volumeOpType) err
 			reqOptions[key] = op.DefaultValue
 		}
 
-	}
-	var volinfo volume.Volinfo
-	if err := c.Get("volinfo", &volinfo); err != nil {
-		return err
+	case xlator.VolumeStart:
+		fallthrough
+	case xlator.VolumeStop:
+		//start the daemon, if daemon option default value is "on",
```

Comment:
The comment is incorrect. The actor framework is generic and is not limited to volume starting or stopping. Please remove this comment.